build.yaml

```diff
@@ -2,7 +2,7 @@ projectType: extension
 additionalTestConfigs:
   jdk11:
     testJdkTool: TEMURIN-JDK11
-    mavenAdditionalArgs: -Dtest=none -DfailIfNoTests=false -DminVersion=4.3.0
+    mavenAdditionalArgs: -Dtest=none -DfailIfNoTests=false -DruntimeVersion="4.4.0-20240215" -DadditionalRuntimeVersions="4.3.0-20231026"
```

Comment:
Why not testing with all versions from 4.3.0?

Reply:
we are -DadditionalRuntimeVersions="4.3.0-20231026"